### ISSUE TITLE
Broken links on user-guide page

### ISSUE THREAD

**Issue by SafeEHA** [metal3-io/metal3-io.github.io#283]
While going through the user-guide for metal3 I came across some broken links, 
- Under 4.1 Install cluster-api metal3 provider [TODO](https://book.metal3.io/capm3/TODO) returns an error when clicked 
- Under 4.3.1 Remediation [Remediation](https://cluster-api.sigs.k8s.io/tasks/healthcheck.html) and [CAPI MHC](https://cluster-api.sigs.k8s.io/tasks/healthcheck.html) an error when clicked 

@hroyrh and @elfosardo I would be happy to assist in fixing this

**Comment by Rozzii** [comment]
@elfosardo I am willing to accept this, should I ?

**Comment by SafeEHA** [comment]
Yes please @Rozzii ...thanks

**Comment by tuminoid** [comment]
I guess the real question is why isn't the broken link checker alerting us about these? Not only the links need to be fixed, but also the link checker action is to be checked.

**Comment by SafeEHA** [comment]
I believe there's something wrong with the checker, thanks @tuminoid for highlighting this.
I would be happy to help in anyway I can @Rozzii

**Comment by SafeEHA** [comment]
Hello @Rozzii  Please I would like to take this up, might need some help if you're available...thanks

**Comment by iamatifmoin** [comment]
@SafeEHA are you still working on this?

**Comment by SafeEHA** [comment]
Yes I am still working on it, I have been having a few challenges while installing which is why I haven't submitted a PR yet

**Comment by SafeEHA** [comment]
Hello, I would like to request the correct links to the broken links mentioned below
.
.
.
> While going through the user guide for metal3 I came across some broken links,
> 
> * Under 4.1 Install cluster-api metal3 provider [TODO](https://book.metal3.io/capm3/TODO) returns an error when clicked
> * Under 4.3.1 Remediation [Remediation](https://cluster-api.sigs.k8s.io/tasks/healthcheck.html) and [CAPI MHC](https://cluster-api.sigs.k8s.io/tasks/healthcheck.html) an error when clicked
> 
> @hroyrh and @elfosardo I would be happy to assist in fixing this

**Comment by okerekechinweotito** [comment]
@SafeEHA 
Have you fixed the issue ?

**Comment by SafeEHA** [comment]
Hello @furkatgofurov7 @fmuyassarov 
Please can you help with these broken links, for instance here, 
4.1 Install cluster-api metal3 provider [TODO](https://book.metal3.io/capm3/TODO) 
I need to know if you're referencing [TODO](https://ironicbaremetal.org/) or something else
cc @elfosardo @Rozzii

**Comment by furkatgofurov7** [comment]
@SafeEHA hi. You can fix/replace any TODO linkn in the user-guide IF the page is available for the action that TODO mentions. 

For example: 

https://book.metal3.io/capm3/installation_guide.html mentions some TODOs:
1. Install Ironic, refer to [TODO](https://book.metal3.io/capm3/TODO) can be replaced with proper link available [here](https://book.metal3.io/ironic/ironic_installation.html)

**Comment by SafeEHA** [comment]
Thanks for your response @furkatgofurov7 
Please I would also like to confirm if this [CAPI MHC](https://cluster-api.sigs.k8s.io/tasks/automated-machine-management/healthchecking.html) is the proper link instead of [this](https://cluster-api.sigs.k8s.io/tasks/healthcheck.html)

**Comment by SafeEHA** [comment]
Hello @Rozzii 
I just created a PR [this](https://github.com/metal3-io/metal3-docs/pull/323) and [this](https://github.com/metal3-io/metal3-docs/pull/324) , please can you review and give me feedback
